examples/remote_exec/program.py

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env python3
+
+import sys
+import os
```

Comment:
Some notes (non-blockers):

PEP 8 would have us order standard library calls alphabetically, and I generally try to ensure every example we put out follows all standard style guidelines to avoid people complaining about trivial things. Same with spaces on line 10 around the plus signs (and, to be honest, I'd probably use an f-string there for readability, but that's more of a preference than a style note).

Reply:
I'll fix the imports and the plug signs (and run the whole thing through `pylint`). I'm avoiding using f-string because ec2 instances have python(2) but not python3. I don't want to make the example run even longer by installing python3.